data/Archlinux.yaml

```diff
@@ -4,11 +4,11 @@ prometheus::server::manage_user: false
 prometheus::server::manage_group: false
 prometheus::server::package_ensure: 'installed'
 prometheus::server::manage_config_dir: false
-prometheus::server::manage_init_file: false
+prometheus::server::manage_init_file: true
 prometheus::configname: 'prometheus.yml'
-prometheus::version: '2.46.0'
-prometheus::install_method: 'package'
-prometheus::bin_dir: '/usr/bin'
+prometheus::server::version: '2.51.2'
```

Comment:
is that even relevant when the install_method is package?

Reply:
It's not. Let's see if I have to adapt the tests ...